### PR TITLE
Add or modify config filess for deploy to ElasticBeanStalk

### PR DESCRIPTION
**Pull request by syuanivy**
pip freeze requirements.txt
    change pythonpath
    update 02_python.config to handle migration
    update db setting in setting.py for DB connection
    add global configuration
    added eb package configuration

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+appdirs==1.4.3
```
Comment (neptscorpio) [inline requirements.txt]: Is this the minimal requirement?

File: .ebextentions/02_python.config [hunk 2]
```diff
@@ -0,0 +1,14 @@
+container_commands:
```
Comment (neptscorpio) [inline .ebextentions/02_python.config]: So these two config files work everywhere, right?

File: requirements.txt (hunk 1 above)
Comment (syuanivy) [inline requirements.txt]: So far yes.

File: .ebextentions/01_packages.config [hunk 3]
```diff
@@ -0,0 +1,5 @@
+packages:
+  yum:
+    git: []
+    postgresql93-devel: []
```
Comment (syuanivy) [inline .ebextentions/01_packages.config]: Now that I think about why we had to do yum install postgresql-devel by ourselves, it seems that this **postgresql93-devel**
didn't work for us. Next time I will try to change it to  **postgresql-devel** and see if it works

File: .ebextentions/02_python.config (hunk 2 above)
Comment (syuanivy) [inline .ebextentions/02_python.config]: They are only relevant during deploy assuming we will keep using aws. AWS will fetch the config it needs from these files.

**Pull request merged**